beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,486 @@ title: Quick Start
+You can put any dynamic expressions inside JSX curlies, including function calls and object literals:
 
 <Sandpack>
 
 ```js
-function Profile({ name, imageUrl }) {
-  return (
-    <img
-      className="avatar"
-      src={imageUrl}
-      alt={name}
-    />
-  );
-}
-
-export default function Gallery() {
+const user = {
+  name: 'Hedy Lamarr',
+  imageUrl: 'https://i.imgur.com/yXOvdOSs.jpg',
+  imageSize: 90,
+};
+
+export default function Profile() {
+  const classes = ['avatar'];
+  if (user.imageSize > 100) {
+    classes.push('large');
+  }
   return (
-    <section>
-      <h1>Amazing scientists</h1>
-      <Profile
-        name="Lin Lanying"
-        imageUrl="https://i.imgur.com/1bX5QH6.jpg"
-      />
-      <Profile
-        name="Gregorio Y. Zara"
-        imageUrl="https://i.imgur.com/7vQD0fPs.jpg"
-      />
-      <Profile
-        name="Hedy Lamarr"
-        imageUrl="https://i.imgur.com/yXOvdOSs.jpg"
+    <>
+      <h1>{user.name}</h1>
+      <img
+        src={user.imageUrl}
+        className={classes.join(' ')}
+        alt={'Photo of ' + user.name}
+        style={{
+          width: user.imageSize,
+          height: user.imageSize
+        }}
       />
-    </section>
+    </>
   );
 }
 ```
 
 ```css
-img { margin: 0 10px 10px 0; height: 90px; }
-.avatar { border-radius: 50%; }
+.avatar {
+  border-radius: 50%;
+}
+
+.large {
+  border: 4px solid gold;
+}
 ```
 
 </Sandpack>
 
-You might wonder why `className="avatar"` uses quotes but `src={imageUrl}` uses curly braces. In JSX, curly braces are like a ["window into JavaScript"](/learn/javascript-in-jsx-with-curly-braces). They let you run a bit of JavaScript right in your markup! So `src={imageUrl}` reads the `imageUrl` prop declared on the first line and passed from the parent `Gallery` component.
+In the above example, `style={{ }}` is not a special syntax, but a regular object inside the JSX curlies. You can use the `style` attribute when your styles depend on JavaScript variables.
 
-In the above example, all the data was written directly in markup. However, you'll often want to keep it separately. Here, the data is kept in an array. In React, you use JavaScript functions like [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to [render lists](/learn/rendering-lists) of things.
+## Conditional rendering {/*conditional-rendering*/}
 
-<Sandpack>
-
-```js App.js
-import { people } from './data.js';
-import Profile from './Profile.js';
+In React, there is no special syntax for writing conditions. Instead, you'll use the same techniques as you use when writing regular JavaScript code. To render a component conditionally, you can use [`if`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else) statement:
```

Comment:
Might be worth mentioning in some way that `if` statements have to be used before a `return`? I guess it's mentioned earlier with the "dynamic expressions in JSX curlies" comment, but it's a common rookie confusion. I know it's one of those "trying not to give too many side details" types of things.

Reply:
Yeah this is tricky because on some level we have to assume the reader knows JS. If they mess up the syntax, hopefully they can copy and/or use references.